perf/istio-install/setup_istio.sh

```diff
@@ -77,11 +77,15 @@ function download_release() {
 }
 
 function install_istioctl() {
-  release=${1:?release folder}
+  release=${1:?missing release path}
   shift
   "${release}/bin/istioctl" manifest apply --skip-confirmation -d "${release}/manifests" "${@}"
 }
 
+function install_telemetry_addons() {
+  kubectl apply -f "${release}/samples/addons" -n istio-system
```

Comment:
I don't think we want to use the demos, at least not prometheus. We should be using production grade deployments, samples is meant for demos only

Reply:
We can use prometheusOperator to more easily deploy prometheus here, similar to what is being added here https://github.com/istio/tools/pull/1012